src/rules/noBadReferenceRule.ts

```diff
@@ -26,10 +26,10 @@ function walk(ctx: Lint.WalkContext<void>): void {
 	for (const ref of sourceFile.referencedFiles) {
 		if (sourceFile.isDeclarationFile) {
 			if (ref.fileName.startsWith("..")) {
-				ctx.addFailureAt(ref.pos, ref.end, Rule.FAILURE_STRING);
+				ctx.addFailure(ref.pos, ref.end, Rule.FAILURE_STRING);
```

Comment:
That's just the way tslint rules are typically written. https://github.com/palantir/tslint/blob/master/src/rules/noNullKeywordRule.ts